rfcs/20210518-prelu-postoperation/README.md

```diff
@@ -0,0 +1,59 @@
+# Introducing PReLU post operation support in oneDNN
+
+## Motivation
+
+This paper https://arxiv.org/abs/1502.01852 introduced PReLU activation
+function. The DIEN model uses PReLu as matmul activation.
+This gets expanded in TF graph.
+![](graph_prelu.png "Graph attention layer")
+
+It becomes a bottleneck for the Attention layer in DIEN. Currently oneDNN
+supports PReLu only as
+[standalone primitive](https://oneapi-src.github.io/oneDNN/dev_guide_prelu.html).
+
+## Proposal
+
+Extend [supported postops list](https://oneapi-src.github.io/oneDNN/dev_guide_attributes_post_ops.html)
+by adding PReLU forward postop.
```

Comment:
I assume this is only for forward inference, but it should still be mentioned

Reply:
Updated.